experimental/SetPartitions/src/AbstractPartition.jl

```diff
@@ -0,0 +1,18 @@
+"""
+AbstractPartition
+
+Abstract type of Partition for classical-, colored- and spatial Partitions.
+"""
+abstract type AbstractPartition end
+
+function composition(p::T, q::T) where {T <: AbstractPartition}
+    composition_loops(p, q)[1]
+end
+
+function ⋅(p::T, q::T) where {T <: AbstractPartition}
+    composition_loops(p, q)[1]
+end
+
+function ⊗(p::T, q::T) where {T <: AbstractPartition}
+    tensor_product(p, q)
+end
```

Comment:
Missing trailing newline

Reply:
Are the missing newlines there? If not, what is the best way to display them? All the other comments so far should haven been resolved.